src/OpenPr0gramm/Response/CaptchaResponse.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+
+namespace OpenPr0gramm
+{
+#if FW
+    [Serializable]
+#endif
+    public class CaptchaResponse : Pr0grammResponse
+    {
+        public string Token { get; private set; }
+        public string Captcha { get; private set; }
```

Comment:
Maybe we can add a function to this class that parses the base64 content and returns the captcha image contents as a `byte[]` or a `Stream`.

Reply:
The problem with the `Captcha` field is it still has the data in the following format: `data:image/png;base64,ABCD...`. If using `Convert.FromBase64String` is to be used, the substring after the comma must be extracted. Just did it exactly that way.